src/main/kotlin/com/exactpro/th2/check1/rule/sequence/SilenceCheckTask.kt

```diff
@@ -148,5 +151,14 @@ class SilenceCheckTask(
             resultEvent.status(Event.Status.FAILED)
                 .name("Check failed: $extraMessagesCounter extra messages were found.")
         }
+        return true
```

Comment:
Maybe use the same approach as other rules
```suggestion
        return super.completeEvent(taskState)
```